common/main/ar_PS.xml

```diff
@@ -17,32 +17,32 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<months>
 					<monthContext type="format">
 						<monthWidth type="abbreviated">
-							<month type="1">↑↑↑</month>
-							<month type="2">↑↑↑</month>
-							<month type="3">↑↑↑</month>
-							<month type="4">↑↑↑</month>
-							<month type="5">↑↑↑</month>
-							<month type="6">↑↑↑</month>
-							<month type="7">↑↑↑</month>
-							<month type="8">↑↑↑</month>
-							<month type="9">↑↑↑</month>
-							<month type="10">↑↑↑</month>
-							<month type="11">↑↑↑</month>
-							<month type="12">↑↑↑</month>
+							<month type="1">كانون الثاني</month>
+							<month type="2">شباط</month>
+							<month type="3">آذار</month>
+							<month type="4">نيسان</month>
+							<month type="5">أيار</month>
+							<month type="6">حزيران</month>
+							<month type="7">تموز</month>
+							<month type="8">آب</month>
+							<month type="9">أيلول</month>
+							<month type="10">تشرين الأول</month>
+							<month type="11">تشرين الثاني</month>
+							<month type="12">كانون الأول</month>
 						</monthWidth>
 						<monthWidth type="narrow">
-							<month type="1">↑↑↑</month>
-							<month type="2">↑↑↑</month>
-							<month type="3">↑↑↑</month>
-							<month type="4">↑↑↑</month>
-							<month type="5">↑↑↑</month>
-							<month type="6">↑↑↑</month>
-							<month type="7">↑↑↑</month>
-							<month type="8">↑↑↑</month>
-							<month type="9">↑↑↑</month>
-							<month type="10">↑↑↑</month>
-							<month type="11">↑↑↑</month>
-							<month type="12">↑↑↑</month>
+							<month type="1">ك</month>
```

Comment:
The code in question is:
```
                            if (isL1) {
                                harden = true;
                                message2 = "; L1";
                            } else {
                                // for all others, we check to see if the parent's lateral value is
                                // the same as ours
                                // If it is, we are ok, since one of that parent's parents will be
                                // hardened

                                if (parentFile == null) {
                                    parentFile = factory.make(parentId, true);
                                }
                                String parentValue = parentFile.getStringValueWithBailey(xpath);
                                if (!baileyValue.equals(parentValue)) {
                                    harden = true; // true if parentValue == null, see comment below
```

Presumably in this case, parentValue and baileyValue differ. I'll check what those values are and report here shortly

Reply:
ar_PS

//ldml/dates/calendars/calendar[@type="gregorian"]/months/monthContext[@type="format"]/monthWidth[@type="narrow"]/month[@type="2"]

baileyValue = ش 

parentValue = ف

Reply:
//ldml/dates/calendars/calendar[@type="gregorian"]/months/monthContext[@type="format"]/monthWidth[@type="narrow"]/month[@type="2"]

inherits laterally from

//ldml/dates/calendars/calendar[@type="gregorian"]/months/monthContext[@type="stand-alone"]/monthWidth[@type="narrow"]/month[@type="2"]

It's evident here that the "stand-alone" value in the sublocale differs from the parent locale:

https://st.unicode.org/cldr-apps/v#/ar_PS/Gregorian/44527752c46f6cd3